docs/source/user_guide/installation/install-from-docker.rst

```diff
@@ -0,0 +1,129 @@
+Install from source (Docker)
+=========================================
+
+.. contents::
+
+This page describes the build and installation of PyBaMM from the source code, available on GitHub. Note that this is **not the recommended approach for most users** and should be reserved to people wanting to participate in the development of PyBaMM, or people who really need to use bleeding-edge feature(s) not yet available in the latest released version. If you do not fall in the two previous categories, you would be better off installing PyBaMM using pip or conda.
+
+Prerequisites
+-------------
+Before you begin, make sure you have Docker installed on your system. You can download and install Docker from the official `Docker website <https://www.docker.com/get-started/>`_.
+Ensure Docker installation by running :
+
+.. code:: bash
+
+	  docker --version
+
+Pulling the Docker Image (will be supported soon)
```

Comment:
Is this not available already?

Reply:
Yes, now all the initial versions of pybamm are available on Docker hub.